test/rulesets/Base/mapreduce.jl

```diff
@@ -165,6 +168,67 @@ struct SumRuleConfig <: RuleConfig{Union{HasReverseMode}} end
             test_rrule(prod, v)
         end
     end # prod
+
+    @testset "foldl(f, ::Array)" begin
+        CFG = ChainRulesTestUtils.ADviaRuleConfig()
```

Comment:
BlueStyle wants this to be lowercase.

Reply:
Or a const at the top of the page? Instead of 4 times.

Reply:
Correct those are the options.